### DISCUSSION TITLE
Taming Transformers not found at path

### DISCUSSION THREAD

**Discussion by jeannen** [TheLastBen/fast-stable-diffusion#302] [General]
Hey! Everytime I try to run the collab, I get this error:

> Warning: Taming Transformers not found at path /content/gdrive/MyDrive/sd/stable-diffusion/src/taming-transformers/taming
> Warning: BLIP not found at path /content/gdrive/MyDrive/sd/stable-diffusion/src/blip/models/blip.py
> Traceback (most recent call last):
>   File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py", line 12, in <module>
>     from modules import devices, sd_samplers, upscaler
>   File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/sd_samplers.py", line 10, in <module>
>     from modules import prompt_parser, devices, processing, images
>   File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/processing.py", line 14, in <module>
>     import modules.sd_hijack
>   File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/sd_hijack.py", line 10, in <module>
>     import modules.textual_inversion.textual_inversion
>   File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/textual_inversion/textual_inversion.py", line 13, in <module>
>     from modules import shared, devices, sd_hijack, processing, sd_models, images
>   File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/shared.py", line 14, in <module>
>     import modules.sd_models
>   File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/sd_models.py", line 13, in <module>
>     from modules.sd_hijack_inpainting import do_inpainting_hijack, should_hijack_inpainting
>   File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/sd_hijack_inpainting.py", line 6, in <module>
>     import ldm.models.diffusion.ddpm
>   File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/ldm/models/diffusion/ddpm.py", line 24, in <module>
>     from ldm.models.autoencoder import VQModelInterface, IdentityFirstStage, AutoencoderKL
>   File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/ldm/models/autoencoder.py", line 6, in <module>
>     from taming.modules.vqvae.quantize import VectorQuantizer2 as VectorQuantizer
> ImportError: cannot import name 'VectorQuantizer2' from 'taming.modules.vqvae.quantize' (/usr/local/lib/python3.7/dist-packages/taming/modules/vqvae/quantize.py)

I tried to add
`
!pip install taming-transformers
!pip install BLIP
`

but it doesn't really change anything, any idea of what should I do?

**Comment by TheLastBen** [comment]
Remove the folder "SD" from your gdrive and do a clean run, make sure you use the latest colab

**Comment by jeannen** [comment]
Amazing, it worked, thanks!! 😄
Do you know if it's possible to use it with SD 1.5?

**Reply by TheLastBen** [reply]
1.5 was added at the moment of its release, make sure you're using the latest Colab : 
https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast_stable_diffusion_AUTOMATIC1111.ipynb

**Comment by jeannen** [comment]
Oh, my bad, it was indeed already added
Thought it was 1.4 because I got bad results compared to other collabs, I guess I need to improve the prompt and everything then 😅
